### ISSUE TITLE
how many epochs do you run for NER tasks? Conll2003 and Wnut?

### ISSUE THREAD

**Issue by omerarshad** [zalandoresearch/flair#269]
Any details for experiments done with wnut  dataset to achieve 50.20? are these results published?

**Comment by alanakbik** [comment]
In the original experiments, we ran for 150 epochs max. In current experiments, we are anneal by a factor of 0.5 and with a patience of 3 until the learning rate becomes too small. In many experiments, that happens after around 100-150 epochs. 

The WNUT experiment is not yet published. We are now running more experiments for the upcoming release-0.4. With this release, we will likely update instructions on how to reproduce numbers and will include WNUT here as well!

**Comment by alanakbik** [comment]
We've just release version 0.4. In the newest version, here is the full code to reproduce WNUT numers: 

```python
from flair.data_fetcher import NLPTaskDataFetcher, NLPTask
from flair.data import TaggedCorpus
from flair.embeddings import  WordEmbeddings, StackedEmbeddings, CharLMEmbeddings

# 1. get the corpus
from flair.trainers.trainer import ModelTrainer

corpus: TaggedCorpus = NLPTaskDataFetcher.load_corpus(NLPTask.WNUT_17)
print(corpus)
# 2. what tag do we want to predict?
tag_type = 'ner'

# 3. make the tag dictionary from the corpus
tag_dictionary = corpus.make_tag_dictionary(tag_type=tag_type)
print(tag_dictionary.idx2item)

embeddings = StackedEmbeddings(embeddings=[
    WordEmbeddings('crawl'),
    WordEmbeddings('twitter'),
    CharLMEmbeddings('news-forward'),
    CharLMEmbeddings('news-backward'),
])

# initialize sequence tagger
from flair.models import SequenceTagger

tagger: SequenceTagger = SequenceTagger(hidden_size=256,
                                        embeddings=embeddings,
                                        tag_dictionary=tag_dictionary,
                                        tag_type=tag_type)
print(tagger)
# initialize trainer
trainer: ModelTrainer = ModelTrainer(tagger, corpus)

results = trainer.train('resources/taggers/wnut-experiment',
                        max_epochs=150,
                        train_with_dev=True)

print(results)

```
